Cargo.toml

```
@@ -92,16 +92,14 @@ bytes = "1"
 clap = { version = "4.5.4", features = ["cargo", "wrap_help"] }
 # Turn off c-kzg's default features which include `blst/portable`. We can turn on blst's portable
 # feature ourselves when desired.
-# TODO(das): switch to c-kzg crate before merging back to unstable (and disable default-features) if possible
-# Turn off c-kzg's default features which include `blst/portable`. We can turn on blst's portable
-# feature ourselves when desired.
-c-kzg = { git = "https://github.com/ethereum/c-kzg-4844", rev = "114fa0382990e9b74b1f90f3b0dc5f97c2f8a7ad" }
+c-kzg = { version = "1", default-features =  false }
 compare_fields_derive = { path = "common/compare_fields_derive" }
 criterion = "0.3"
 delay_map = "0.3"
 derivative = "2"
 dirs = "3"
 either = "1.9"
+eip7594 = { git = "https://github.com/crate-crypto/peerdas-kzg", rev = "5a44c5995ade5de0ac06909bcbacab8437847ba4" }
```

Why not name the lib peerdas-kzg?

---

The name is going to change in the future, it is called eip7594 for historical reasons -- the "finalized" name will include the word "eth", likely "rust-eth-kzg"

---

Perhaps we could rename it here for clarity within the Lighthouse repo?

```suggestion
peerdas-kzg = { git = "https://github.com/crate-crypto/peerdas-kzg", rev = "5a44c5995ade5de0ac06909bcbacab8437847ba4", package = "eip7594" }
```